addons/mail/static/src/core/common/message_model.js

```diff
@@ -189,7 +189,7 @@ export class Message extends Record {
     }
 
     get isSubjectDefault() {
-        const threadName = this.originThread?.name?.trim().toLowerCase();
+        const threadName = this.originThread?.name ? this.originThread?.name?.trim().toLowerCase() : "";
```

Comment:
That one is no longer necessary then.
```suggestion
        const threadName = this.originThread?.name ? this.originThread?.name.trim().toLowerCase() : "";
```